Dockerfile-dc

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.10
```

Comment:
Use `golang:1.13-alpine` instead. You don't need `golang:latest` then

Reply:
Alright, will change that!